index.html

```diff
@@ -313,63 +317,73 @@ <h1>Get Started</h1>
         </div>
         <!-- end mobile get started version -->
 
-        <div class="imagegeometry6">
-            <img src="images/image-geometry_6.svg" alt="">
-        </div>
+        <div class="get-started">
+            <div class="imagegeometry6">
+                <img src="images/image-geometry_6.svg" alt="">
+            </div>
 
-        <div class="started-tit">
-            <h1>Get Started</h1>
-            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, 
-                quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
-        </div>
+            <div class="started-tit">
+                <h1>Get Started</h1>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, 
+                    quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+            </div>
 
-        <div class="imagegeometry7">
-            <img src="images/image-geometry_7.svg" alt="">
-        </div>
+            <div class="imagegeometry7">
+                <img src="images/image-geometry_7.svg" alt="">
+            </div>
 
-        <div id="get-started" class="form">
-            <form action="https://formspree.io/f/myyldona" method="POST">
-                <div class="formcontainer">
-                    <div class="fistname">
-                        <label for="fname">First Name</label>
-                        <br>
-                        <input type="text" id="fname" name="firstname" maxlength="30" required placeholder="Your name..">
-                    </div>
-                    <div class="email">
-                        <label for="email">E-mail</label>
-                        <br>               
-                        <input type="email" id="email" name="_replyto" required placeholder="Your e-mail..">
+            <div id="get-started" class="form">
+                <form action="https://formspree.io/f/myyldona" method="POST">
+                    <div class="formcontainer">
+                        <div class="fistname">
+                            <label for="fname">First Name</label>
+                            <br>
+                            <input type="text" id="fname" name="firstname" maxlength="30" required placeholder="Your name..">
+                        </div>
+                        <div class="email">
+                            <label for="email">E-mail</label>
+                            <br>               
+                            <input type="email" id="email" name="_replyto" required placeholder="Your e-mail..">
+                        </div>
```

Comment:
- [ ] Kindly, fix the HTML or Page Overflow-x, make it overflow-x: hidden to fix that.